src/core/reference.js

```diff
@@ -0,0 +1,344 @@
+/**
+ * @module Foundation
+ * @submodule Declaration
+ * @for p5
+ */
+
+
+/**
+
+ * Declares a block scope local variable, optionally initializing it to a value.
+ *
+ * @property let
+ * 
+ * Syntax:
```

Comment:
Explanatory text needs to go above the `@property, @method, etc` line in order to appear.